src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -97,63 +163,51 @@ export const SpendReport = ({ tab, billingProject }) => {
+                compute: parseFloat(_.find(['category', 'Compute'], categorySpendData)?.cost ?? '0'),
+                storage: parseFloat(_.find(['category', 'Storage'], categorySpendData)?.cost ?? '0'),
+                other: parseFloat(_.find(['category', 'Other'], categorySpendData)?.cost ?? '0')
+              }
             }
+            const costDict = getCategoryCosts(categoryDetails.spendData)
 
-            // @ts-ignore
-            setProjectCost(totalCosts)
+            setProjectCost({
+              spend: costFormatter.format(parseFloat(spend.spendSummary.cost)),
+              compute: costFormatter.format(costDict.compute),
+              storage: costFormatter.format(costDict.storage),
+              other: costFormatter.format(costDict.other)
+            })
 
-            // @ts-ignore
-            const workspaceDetails = _.find(details => details.aggregationKey === 'Workspace')(spend.spendDetails)
+            const workspaceDetails = _.find(details => details.aggregationKey === 'Workspace', spend.spendDetails) as AggregatedWorkspaceSpendData
             console.assert(workspaceDetails !== undefined, 'Spend report details do not include aggregation by Workspace')
             // Get the most expensive workspaces, sorted from most to least expensive.
-            // @ts-ignore
             const mostExpensiveWorkspaces = _.flow(
               _.sortBy(({ cost }) => { return parseFloat(cost) }),
               _.reverse,
-              _.slice(0, maxWorkspacesInChart) // @ts-ignore
-            )(workspaceDetails?.spendData)
+              _.slice(0, maxWorkspacesInChart)
+            )(workspaceDetails?.spendData) as WorkspaceSpendData[]
             // Pull out names and costs.
-            const costPerWorkspace = {
-              // @ts-ignore
+            const costPerWorkspace: WorkspaceCosts = {
               workspaceNames: [], computeCosts: [], storageCosts: [], otherCosts: [], costFormatter, numWorkspaces: workspaceDetails?.spendData.length
             }
             _.forEach(workspaceCostData => {
-              // @ts-ignore
               costPerWorkspace.workspaceNames.push(workspaceCostData.workspace.name)
-              // @ts-ignore
               const categoryDetails = workspaceCostData.subAggregation
-              console.assert(categoryDetails.key !== 'Category', 'Workspace spend report details do not include sub-aggregation by Category')
-              const costDict = getCategoryCosts(categoryDetails.spendData, true) // @ts-ignore
-              costPerWorkspace.computeCosts.push(costDict.Compute) // @ts-ignore
-              costPerWorkspace.storageCosts.push(costDict.Storage) // @ts-ignore
-              costPerWorkspace.otherCosts.push(costDict.Other) // @ts-ignore
-            })(mostExpensiveWorkspaces) // @ts-ignore
+              console.assert(categoryDetails.aggregationKey === 'Category', 'Workspace spend report details do not include sub-aggregation by Category')
```

Comment:
A bug! I was incorrectly checking `category.key` and also had the assert reversed (so it always passed).